### PR TITLE
add disabled button style

### PR DESCRIPTION
**Pull request by SijiaDavis**
add disabled style to the style mixin. also added the demo.

### CODE REVIEW

File: demo/mixins.html [hunk 1]
```diff
@@ -159,5 +159,33 @@ <h3>Pill Button - ll-button demo</h3>
...
+            import {buttonDisabled} from '../ll-button-style-mixins.js';
+            class DisabledStyledButton extends LitElement {
+              render() {
+                return html`
+                    <style>
+                      :host {
+                        ${buttonDisabled}
+                      }
+                    </style>
+                    <ll-button text="Disabled Button Demo"></ll-button>
```
Comment (Westbrook) [inline demo/mixins.html]: Is it overkill to be explicit with this usecase?
```suggestion
                    <ll-button disabled text="Disabled Button Demo"></ll-button>
```
Which would change a bit the styles above as well?

File: ll-button-style-mixins.js [hunk 2]
```diff
@@ -61,3 +61,15 @@ export const buttonPill = html`
     calc(var(--ll-rhythm, 10px) * 0.5)
     calc(var(--ll-rhythm, 10px) * 2);
 `;
+
+export const buttonDisabled = html`
```
Comment (Westbrook) [inline ll-button-style-mixins.js]: Another option for being explicit would be to make this be something like:
```
ll-button[disabled] {
  ...
}
``` But that's probably taking it too far.

Comment (SijiaDavis) [reply ll-button-style-mixins.js]: based on an offline discussion we will hold it off until we can get Hayashi's input on this

File: demo/mixins.html [hunk 3]
```diff
@@ -159,5 +159,33 @@ <h3>Pill Button - ll-button demo</h3>
...
+        <template>
+          <disabled-styled-button></disabled-styled-button>
+          <script type="module">
+            import {LitElement, html} from '../node_modules/@polymer/lit-element/lit-element.js';
+            import {buttonDisabled} from '../ll-button-style-mixins.js';
+            class DisabledStyledButton extends LitElement {
+              render() {
+                return html`
+                    <style>
+                      :host {
```
Comment (Westbrook) [inline demo/mixins.html]:
```suggestion
                      [disabled] {
```

**Pull request merged**